doc/advanced.rst

```diff
@@ -400,16 +400,21 @@ Most of the time though, a tag is not needed:
       {{ '**markdown** text'|markdown }}
 
   If you want use this filter on large amounts of text, wrap it with the
-  :doc:`filter <tags/filter>` tag:
+  :doc:`apply <tags/apply>` tag:
 
   .. code-block:: jinja
 
-      {% filter markdown %}
+      {% apply markdown %}
       Title
       =====
 
       Much better than creating a tag as you can **compose** filters.
-      {% endfilter %}
+      {% endapply %}
+
+ .. note::
+
+      The ``apply`` tag was introduced in Twig 1.40; use the ``filter`` tag with
+      previous versions.
```

Comment:
I would add a note about `but be careful that it introduces a new scope for variables`.

Reply:
Not sure it's relevant here as I added a note about this in the `filter`docs and nobody was really aware before.